ion/textreader.go

```diff
@@ -183,6 +187,16 @@ func (t *textReader) nextBeforeFieldName() (bool, error) {
 			}
 		}
 
+		if tok == tokenSymbolQuoted {
+			t.fieldNameSymbol = SymbolToken{Text: &val, LocalSID: SymbolIDUnknown}
```

Comment:
It's hard to tell at a glance without an IDE: is the value of `SymbolIDUnknown` zero?

Reply:
`SymbolIDUnknown` has value `-1`